packages/EasyCodingStandard/src/Finder/SourceFinder.php

```diff
@@ -26,43 +25,14 @@ public function find(array $source): array
     {
         $files = [];
 
-        foreach ($source as $singleSource) {
-            if (is_file($singleSource)) {
-                $files = $this->processFile($files, $singleSource);
-            } else {
-                $files = $this->processDirectory($files, $singleSource);
-            }
+        if (! $this->extraFilesProvider) {
+            $this->extraFilesProvider[] = new DefaultFilesProvider;
```

Comment:
Tests are probably false positvites, this is probably never run.
Could you add a to ensure it is? 

Also I'd move it constructor injection, to prevent others adding static to the code and allow switch fro others if needed.